condor_SUEP_WS.py

```diff
@@ -1,73 +1,73 @@
+elif options.analyzer == "ZH_simple_withsyst":
+  from workflows.SUEP_coffea_ZH_simple import *
+  modules_era.append(SUEP_cluster(isMC=options.isMC, era=int(options.era), do_syst=1,  syst_var='', sample=options.dataset, weight_syst='' , SRonly=options.SR, output_location=out_dir, doOF=False, isDY=options.isDY))
+elif options.analyzer == "ZH_simple_fortests":
+  from workflows.SUEP_coffea_ZH_simple_fortests import *
+  modules_era.append(SUEP_cluster(isMC=options.isMC, era=int(options.era), do_syst=0,  syst_var='', sample=options.dataset, weight_syst='' , SRonly=options.SR, output_location=out_dir, doOF=False, isDY=options.isDY))
+elif options.analyzer == "ZH_simple_OF":
+  from workflows.SUEP_coffea_ZH_simple import *
+  modules_era.append(SUEP_cluster(isMC=options.isMC, era=int(options.era), do_syst=0,  syst_var='', sample=options.dataset, weight_syst='' , SRonly=options.SR, output_location=out_dir, doOF=True))
+elif options.analyzer == "ZH_trackID":
+  from workflows.SUEP_coffea_ZH_trackID import * 
+  modules_era.append(SUEP_cluster(isMC=options.isMC, era=int(options.era), do_syst=0,  syst_var='', sample=options.dataset, weight_syst='' , flag=False, output_location=out_dir))
+elif options.analyzer == "ZH_trackProps":
+  from workflows.SUEP_coffea_ZH_trackProps import * 
+  modules_era.append(SUEP_cluster(isMC=options.isMC, era=int(options.era), do_syst=0,  syst_var='', sample=options.dataset, weight_syst='' , flag=False, output_location=out_dir))
+elif options.analyzer == "ZH_gen":
+  from workflows.SUEP_coffea_ZH_onlygenZpt import * 
+  modules_era.append(SUEP_cluster(isMC=options.isMC, era=int(options.era), do_syst=0,  syst_var='', sample=options.dataset, weight_syst='' , flag=False, output_location=out_dir, doOF=False, isDY=options.isDY))
+elif options.analyzer == "ZH_leptonID":
+  from workflows.SUEP_coffea_ZH_leptonID import * 
+  modules_era.append(SUEP_cluster(isMC=options.isMC, era=int(options.era), do_syst=0,  syst_var='', sample=options.dataset, weight_syst='' , SRonly=options.SR, output_location=out_dir, doOF=True, isDY = options.isDY))
+elif options.analyzer == "ZH_btagEff":
+  from workflows.SUEP_coffea_ZH_bTagEff import *
+  modules_era.append(SUEP_cluster(isMC=options.isMC, era=int(options.era), do_syst=0,  syst_var='', sample=options.dataset, weight_syst='' , SRonly=options.SR, output_location=out_dir, doOF=True, isDY = options.isDY))
+
+for instance in modules_era:
+    output = run_uproot_job(
+        {instance.sample: [options.infile]},
+        treename='Events',
+        processor_instance=instance,
+        executor=futures_executor,
+        executor_args={'workers': 1,
+                       'schema': processor.NanoAODSchema,
+                       'xrootdtimeout': 10,
+        },
+        chunksize = 100 if options.test else options.chunksize,
+        maxchunks = 1 if options.test else None,
+    )

```

Comment:
Please don't do this, if you don't change anything on a file, don't commit it full again. It makes review harder and then it is more difficult to track changes